Gemfile

```diff
@@ -28,7 +28,7 @@ group :test do
   gem 'redcarpet',                                                  :require => false
   gem 'rubocop', '~> 0.49.1',                                       :require => false
   gem 'rubocop-rspec', '~> 1.15.0',                                 :require => false
-  gem 'mocha', '~> 1.4.0',                                          :require => false
+  gem 'mocha', '~> 1.10.0',                                         :require => false
```

Comment:
The Gemfile content will be overridden by the next modulesync unless you customise its behaviour by modifying `.sync.yml` too.  See https://github.com/voxpupuli/puppet-redis/blob/master/.sync.yml#L10 for example.